x-pack/plugins/stack_alerts/server/alert_types/es_query/alert_type.ts

```diff
@@ -24,7 +26,7 @@ export function getAlertType(
   core: CoreSetup
 ): RuleType<
   EsQueryAlertParams,
-  never, // Only use if defining useSavedObjectReferences hook
+  EsQueryAlertParamsExtractedParams, // Only use if defining useSavedObjectReferences hook
```

Comment:
now that it is used, think we no longer need this remark
```suggestion
  EsQueryAlertParamsExtractedParams,
```